test/mechanics/mesh/UnitMeshFem.cpp

```diff
@@ -28,6 +28,36 @@ void Check2DMesh(NuTo::MeshFem& mesh)
     BOOST_CHECK_NO_THROW(transformedMesh.NodeAtCoordinate(Eigen::Vector2d(43., 9.)));
 }
 
+BOOST_AUTO_TEST_CASE(MeshTrusses)
+{
+    auto mesh = NuTo::UnitMeshFem::CreateTrusses(13);
+    BOOST_CHECK_EQUAL(mesh.Elements.Size(), 13);
+    BOOST_CHECK_EQUAL(mesh.Nodes.Size(), 14);
+
+    bool nodesHaveExpectedValue = true;
+    for (unsigned int i = 0; i < mesh.Nodes.Size(); ++i)
```

Comment:
Your test code is _very_ similar to the actual implementation. So if you have a bug in the implementation, chances are that you have the same bug in the test. This may not be a problem in this simple case, but generally I prefer testing against hardcoded values.